server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestAnalyzeIndexDiskUsageAction.java

```diff
@@ -25,6 +27,7 @@
 
 import static org.elasticsearch.rest.RestRequest.Method.POST;
 
+@ServerlessScope(Scope.PUBLIC)
```

Comment:
Is this public or internal?

Reply:
as per our discussion during the sync and notes left afterwards it should be public
```
Useful internally as well as for users in combination with field_usage_stats
```

Reply:
Ah right